### PR TITLE
Explain custom nonce life time

### PR DESCRIPTION
**Pull request by ozh**

### CODE REVIEW

File: src/development/form-security.md [hunk 1]
```diff
@@ -65,3 +65,18 @@ function joe-plugin-delete-everything() {
...
+## Define custom nonce lifetime
+
+By default, nonces are valid 12 hours. From [the source](https://github.com/YOURLS/YOURLS/blob/1.8.2/includes/Config/Config.php#L192-L194):
+```php
+// life span of a nonce in seconds
+if (!defined( 'YOURLS_NONCE_LIFE' ))
+    define( 'YOURLS_NONCE_LIFE', 43200 ); // 3600 * 12
+```
+
+If you want to enforce a different time span, for instance an hour (3600 seconds) you can add the following to your `config.php` :
```
Comment (LeoColomb) [inline src/development/form-security.md:78]:
```suggestion
If you want to enforce a different time span, for instance an hour (3600 seconds) you can add the following to your `config.php`:

```

File: src/development/form-security.md [hunk 2]
```diff
@@ -65,3 +65,18 @@ function joe-plugin-delete-everything() {
     // Now delete everything
     ...
 ```
+
+## Define custom nonce lifetime
+
+By default, nonces are valid 12 hours. From [the source](https://github.com/YOURLS/YOURLS/blob/1.8.2/includes/Config/Config.php#L192-L194):
+```php
```
Comment (LeoColomb) [inline src/development/form-security.md:72]:
```suggestion

```php
```

**Review by LeoColomb — CHANGES_REQUESTED** [review]

File: src/development/form-security.md (hunk 1 above)
Comment (ozh) [reply src/development/form-security.md]: I'll never get used to this English punctuation rule ... :)

File: src/development/form-security.md [hunk 3]
```diff
@@ -65,3 +65,20 @@ function joe-plugin-delete-everything() {
...
+if (!defined( 'YOURLS_NONCE_LIFE' ))
+    define( 'YOURLS_NONCE_LIFE', 43200 ); // 3600 * 12
+```
+
+If you want to enforce a different time span, for instance an hour (3600 seconds) you can add the following to your `config.php`:
+
+```php
+define( 'YOURLS_NONCE_LIFE', 3600 );
+```
+
```
Comment (LeoColomb) [inline src/development/form-security.md:84]:
```suggestion
```

Oh and this EOF extra line 😅

**Review by LeoColomb — CHANGES_REQUESTED** [review]

**Review by LeoColomb — APPROVED** [review]

**Pull request merged**